### DISCUSSION TITLE
"Vaccinated Person" as a Risk Profile

### DISCUSSION THREAD

**Discussion by sparrow925** [microCOVID/microCOVID#782] [Ideas]
Thanks for building and maintaining this incredibly useful tool!

Are there any plans to include "Vaccinated Person" as a Risk Profile? I imagine there are many limitations to adding this so early into the roll-out (data availability, for one), but is it anticipated as a possibility eventually?

**Comment by blanchardjeremy** [comment]
Hi @sparrow925  -- That feature exists now! You can select their vaccine after you select one of the "risk profile" options.
